lib/schemes/flat.js

```diff
@@ -6,5 +6,15 @@ module.exports = {
         var naming = bemNaming(options.naming);
 
         return naming.stringify(entity) + '.' + tech;
+    },
+    parse: function(path, options) {
+        options || (options = {});
+        var naming = bemNaming(options.naming),
+            splittedPath = path.split('.');
```

Comment:
Can we check here that path is valid entity file? At least, can we get `path.basename`?